extensions/2.0/Vendor/EXT_lights_image_based/schema/light.schema.json

```diff
@@ -0,0 +1,68 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "light",
+    "type": "object",
+    "description": "An image-based lighting environment.",
+    "allOf" : [ { "$ref" : "../../../../specification/2.0/schema/glTFChildOfRootProperty.schema.json" } ],
```

Comment:
I know this just got merged, but I need to ask one more question.  None of our other extensions put this whole relative path on here, they all just say `glTFChildOfRootProperty.schema.json` without the path.  Would it be OK to remove the path for consistency?

Reply:
Further discussion migrated to #1480.

Reply:
Sure. I can make that change and do another pull request. Thanks.

Reply:
@MiiBond I needed it for VSCode, so I made the change in #1480.  Can you take a quick look at that?  Thanks!